students/JaeKim/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from customers_model import Customer, TestCustomer
+import customers_model as cm
+
+
+def add_customer(customer_id, name, lastname, home_address, phone_number, email_address, status, credit_limit,
+                 stage='prod'):
+    """
+    :param customer_id:
+    :param name:
+    :param lastname:
+    :param home_address:
+    :param phone_number:
+    :param email_address:
+    :param status:
+    :param credit_limit:
+    :param stage:
+    :return:
+    """
```

Comment:
Cool, nice doc string. Could also include any limitations on the data (status True/False or no $ in money. Whatever type you want for each variable.